apps/web/src/components/RightSideBarActivities.tsx

```diff
@@ -82,8 +84,14 @@ export const RightSideBarActivites: React.FC = () => {
               filteredData &&
               filteredData.map((item: any) => {
                 return (
-                  <div key={item.id}>
-                    <Checkbox name='careActivities' value={item.id} label={item.name} />
+                  <div key={item.id} className='flex flex-1 items-center'>
+                    <div className='flex-initial w-5/6'>
+                      <Checkbox name='checked' value={item.id} label={item.name} />
```

Comment:
I think I've updated the name to careActivities. Kindly update and test the flow

Reply:
Updated to current develop